synapse/rest/media/v1/media_repository.py

```diff
@@ -506,14 +522,35 @@ async def generate_local_exact_thumbnail(
 
             return output_path
 
+        # Could not generate thumbnail.
+        return None
+
     async def generate_remote_exact_thumbnail(
-        self, server_name, file_id, media_id, t_width, t_height, t_method, t_type
-    ):
+        self,
+        server_name: str,
+        file_id: str,
+        media_id: str,
+        t_width: int,
+        t_height: int,
+        t_method: str,
+        t_type: str,
+    ) -> Optional[str]:
         input_path = await self.media_storage.ensure_media_is_in_local_cache(
             FileInfo(server_name, file_id, url_cache=False)
         )
 
-        thumbnailer = Thumbnailer(input_path)
+        try:
+            thumbnailer = Thumbnailer(input_path)
+        except ThumbnailError:
+            logger.warning(
+                "Unable to generate a thumbnail for remote media %s from %s using a method of %s and type of %s",
+                media_id,
+                server_name,
+                t_method,
+                t_type,
+            )
```

Comment:
Do we want log why as well?

Reply:
Hmm....that's a good thought, we'd have to pass the original exception through (I guess using `raise from`, maybe?)

Reply:
@erikjohnston I'm hoping b5b96ea65731f3eb86d5c06b7defd1ab3c23d004 is what you were thinking?